kbase-extension/static/kbase/js/common/cellComponents/fieldTableCellWidget.js

```diff
@@ -5,16 +5,8 @@ define([
     'common/events',
     'common/runtime',
     'widgets/appWidgets2/errorControl',
-    'css!google-code-prettify/prettify.css'
```

Comment:
prettier got in here and did it's thing. i'll try to comment where i made actual changes so it's more clear